README.md

```diff
@@ -28,6 +28,8 @@ Watch a [1-minute video demo](https://www.youtube.com/watch?v=NwqlyBAxVpQ&featur
 gem 'packwerk'
 ```
 
+_Note: Packwerk has to be grouped in production environment as it contains logic to load custom inflectors._
```

Comment:
I’m too late for this PR, but this advice is only true if the app uses custom inflections, which I think we should be clear on so that people don’t add it as a production dependency unnecessarily. My guess would be that most Rails apps don’t configure any custom inflections.

Reply:
In fact, I think this advice should live in the [Inflections section](https://github.com/Shopify/packwerk/blob/main/USAGE.md#inflections) of `USAGE.md`, since it’s adding that initializer which creates the production dependency on `Packwerk::Inflections::Custom`.

Reply:
@wildmaples can you follow up on this? If you have already, ignore me